xgi/classes/hypergraph.py

```diff
@@ -935,6 +936,77 @@ def clear_edges(self):
         self._edge.clear()
         self._edge_attr.clear()
 
+    def merge_duplicate_edges(
+        self, rename="first", merge_rule="first", multiplicity=True
+    ):
+        """Merges edges which have the same members.
+
+        Parameters
+        ----------
+        rename : str, optional
+            Either "first", "tuple", or "new", by default "first"
+            If "first", the new edge ID is the first of the sorted
+            duplicate edge IDs. If "tuple", the new edge ID is a 
+            tuple of the sorted duplicate edge IDs. If "new", the
+            _edge_uid counter is selected and incremented.
+        merge_rule : str, optional
+            Either "first" or "union", by default "first"
+            If "first", takes the attributes of the first duplicate.
+            If "union", takes the set of attributes of all the duplicates.
```

Comment:
Not fully clear to me what the result looks like explicitly. Something like `"color": {"red", "blue"}`?
Maybe examples would be useful.

Reply:
Okay, I added a lot to the docstring...hopefully this addresses this?